manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "User activity watcher",
+  "description": "Watches user activity on issues, sends reminders on deadlines, and unassign inactive users.",
+  "ubiquity:listeners": [ "push", "pull_request.opened", "pull_request.reopened" ]
```

Comment:
Should be on comments because those are the most frequent events.

Reply:
I can make it just for comments then, will change.